tests/functions/test_alias.py

```diff
@@ -32,7 +34,7 @@ def test_alias_boundary():
 
     req = OriginRequest(conf_file=TEST_CONF)
 
-    aliases = [{"src": "/foo/bar", "dest": "/"}]
+    aliases = [{"src": "/foo/bar", "dest": "/", "exclude_paths": []}]
```

Comment:
Please be sure to test alias resolution for aliases without an `exclude_paths` key.